{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/urls.py

```diff
@@ -1,4 +1,4 @@
-"""URL config for {{cookiecutter.repo_name}} project."""
+'''URL config for {{cookiecutter.repo_name}} project.'''
```

Comment:
I was thinking the same.

Can the same be done for `wsgi.py` too? That and the NewRelic archaeological find..